### ISSUE TITLE
Is it possible to make gmrender to listen only on localhost?

### ISSUE THREAD

**Issue by Kafkalasch** [hzeller/gmrender-resurrect#87]

**Comment by hzeller** [comment]
There is the option --ip-address or -I, did you try to set that to 127.0.0.1 ?
(away from computer, have not tested this)

**Comment by Kafkalasch** [comment]
Thank you for your fast response.

Seems to work or at least the logfile does not show any errors. But didn't solve my problem. I am using BubbleUPnP to create an OpenHome renderer out of the gmediarenderer. But doing so there are two renderers listed (i.e. MyPi and MyPi (OpenHome)). If i make the gmediarenderer to listen only on localhost, somehow the BubbleUPnP Server, which also runs on the pi, won't find the gmediarenderer any more. Do you have any suggestions how to solve this?

**Comment by hzeller** [comment]
Oh, if you want to run multiple renderers, you might want to create a
different UUID so that they are possible to distinguish

See
https://github.com/hzeller/gmrender-resurrect/blob/master/INSTALL.md#-u---uuid

**Comment by Kafkalasch** [comment]
oh no, that is not the problem, they are distinguishable. I am just a little perfectionist: I want to make the gmediarenderer invisible to my guests in the network, so that they have to use the OpenHome-Renderer and do not use the gmediarenderer unintenionally. (OpenHome allows many users to create one shared playlist - But UPnPBubbleServer needs the gmediarenderer to create the OpenHome-renderer, it works like an add-on). But never mind. Just wondered if there was an easy solution.

**Comment by hzeller** [comment]
Mmh, so I have not played with BubbleUPnP Server to know what it does (or not does) to only find things on localhost. Did you figure it out ? I'd contact the BubbleUPnP people, I found them to be pretty responsive.

**Comment by Kafkalasch** [comment]
No, I let it be and moved on to other, more pressing features. But thanks for the response and thanks for your hard work on this project!
